_config.yml

```diff
@@ -19,8 +19,8 @@ keywords:    "coala, review, analysis, static, bears"
 description: "coala provides a common command-line interface for linting and fixing all your code, regardless of the programming languages you use."
 source_link: "https://github.com/coala-analyzer/coala/"
 favicon:     "https://raw.githubusercontent.com/coala-analyzer/coala-artwork/master/artwork/coala_bear.png"
-logo:        "http://api.coala.io/en/latest/_static//images/coala_logo.svg"
-
+logo:        "img/coala_logo.png"
```

Comment:
The SVG has a white background?  So, can the SVG be fixed to have a transparent background?